src/renderer/RgbRenderer.py

```diff
@@ -59,7 +59,10 @@ def run(self):
             self._configure_renderer(use_denoiser=True, default_denoiser="Intel")
 
             # In case a previous renderer changed these settings
-            bpy.context.scene.render.image_settings.color_mode = "RGB"
+            #Store as RGB by default unless the user specifies store_alpha as true in yaml
+            bpy.context.scene.render.image_settings.color_mode = "RGBA" if self.config.get_bool("render_alpha", False) else "RGB"
```

Comment:
New config values have to be explained at the top of the class, to make the use for everyone easy.

Reply:
Thanks for modifying, no worries, this PR is great! :) 

I just wanted to make sure that the next user understands it as clearly as you intended it to be.